SimPEG/regularization/base.py

```diff
@@ -560,33 +1044,174 @@ def _cell_distances(self):
 
     @property
     def orientation(self):
+        """Direction along which smoothness is enforced.
+
+        Returns
+        -------
+        str
+            The direction along which smoothness is enforced. On of {'x','y','z'}
```

Comment:
Small typo
```suggestion
            The direction along which smoothness is enforced. One of {'x','y','z'}
```

Reply:
The options should be in the return type. Just {'x','y','z'}. Made the switch.